tests/vmi_configuration_test.go

```diff
@@ -2110,4 +2110,21 @@ var _ = Describe("Configurations", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	It("VMI with masquerade binding and guest agent should expose Pod IP as its public address", func() {
```

Comment:
We moved the MAC address reporting under a condition. Can you create a similar test for that?

Reply:
Added within the same test